### DISCUSSION TITLE
Problem installing Content Builder after reinstalling Visual Studio

### DISCUSSION THREAD

**Discussion by EzraSidran** [MonoGame/MonoGame#8610] [General]
I had to reinstall Visual Studio Microsoft Visual Studio Community 2022 (64-bit) - Current17.12.1. I installed the Monogame Content Builder:
![image](https://github.com/user-attachments/assets/b78a5737-65da-49ea-973a-371aace84065)

But, it doesn't appear like it used to in the content folder:
![image](https://github.com/user-attachments/assets/eefa3670-ec81-43ca-9cec-0f22d99a8630)

Double-clicking on content.mgcb just launches a text file, not the editor like I was expecting. So, I can't product xnb files, right? Any ideas on how to install the editor (I thought of it as a compiler, it makes XNB files, right?) that was before?

**Comment by MonogameGuy** [comment]
The .mgcb text file will always be "compiled" upon build.

Open a command prompt within the VS project folder and run  "dotnet tool list" :

C:\project\dotnet tool list
Package Id                      Version         Commands                 Manifest                                                       
----------------------------------------------------------------------------------------------------------------------------------------
dotnet-mgcb                     3.8.2.1105      mgcb                 
dotnet-mgcb-editor              3.8.2.1105      mgcb-editor
dotnet-mgcb-editor-windows      3.8.2.1105      mgcb-editor-windows

Type:
" dotnet tool run dotnet-mgcb-editor Content/Content.mgcb"

The path is relative to the current project directory containing a "Content" folder.

**Reply by EzraSidran** [reply]
Thanks. I discovered that after I reinstalled VS I had to follow these instructions to 'migrate' to the new version: [Migrating from 3.8.0 ](https://docs.monogame.net/articles/migration/migrate_38.html)
